support-services/src/main/scala/com/gu/helpers/WebServiceHelper.scala

```diff
@@ -11,9 +11,23 @@ import scala.collection.immutable.Map.empty
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 import scala.reflect.{ClassTag, classTag}
+import scala.util.{Failure, Success, Try}
 
-case class WebServiceHelperError[T: ClassTag](responseCode: Int, responseBody: String) extends Throwable {
-  override def getMessage: String = s"${classTag[T]} - $responseCode: $responseBody"
+sealed abstract class WebServiceHelperErrorBase(responseCode: String, responseBody: String) extends Throwable
+
+case class WebServiceHelperError[T: ClassTag](
+  responseCode: String,
```

Comment:
in reality is isn't really a String or an Int, but I changed to string so I can index into it easily